data/eos-companion-app.service

```diff
@@ -6,7 +6,29 @@ ConditionPathExists=/etc/avahi/services/companion-app.service
 [Service]
 ExecStart=/usr/bin/dbus-run-session /usr/bin/flatpak run --no-desktop --no-a11y-bus --no-documents-portal com.endlessm.CompanionAppService
 User=companion-app-helper
+# This does not appear to cause any problems when writing
+# to /var/lib/eos-companion-app, remove if it does. We need
+# CAP_SETUID and CAP_SETGID as bwrap is setuid
+CapabilityBoundingSet=CAP_NET_ADMIN CAP_SYS_ADMIN CAP_SYS_CHROOT CAP_SETUID CAP_SETGID
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateDevices=yes
+# Breaks xapian-bridge
+#PrivateNetwork=yes
 PrivateTmp=yes
+PrivateUsers=yes
+ProtectControlGroups=yes
+ProtectHome=yes
+ProtectKernelModules=yes
+# Breaks bwrap
+#ProtectKernelTunables=yes
+ProtectSystem=yes
+# xapian-bridge needs AF_INET, we should probably
+# allow AF_INET6 too
+RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
```

Comment:
In https://github.com/endlessm/eos-updater/commit/fc03bd7be598d0a3114cb56e1b40113fca2a8f2e, Rob said that bwrap needed `AF_NETLINK`. Is that not the case?

Reply:
It appears not to. Perhaps that is only required when installing flatpaks.

Reply:
Oh weird, it is complaining about netlink now:

Jul 16 16:11:08 endless dbus-run-session[67331]: bwrap: loopback: Failed to create NETLINK_ROUTE socket: Address family not supported by protocol